recipe/meta.yaml

```diff
@@ -9,24 +9,20 @@ package:
 source:
   fn: {{ filename }}
   url: https://github.com/{{ name }}/{{ name }}/archive/{{ filename }}
-  sha256: bb43216b075b93b767e6cbaecd5e95d087da887bcdc51afd36d940d8bd9f9819
+  sha256: bdbf9a2e6988526bfeb8c56ce3cdfe2998d60ac289078e2215374288185e8c8d
 
 build:
   number: 0
   # We skip s390x because it's not supported.
-  skip: True # [s390x]
-  # py-boost hasn't been built for Python 3.11 yet.
-  skip: True # [py<37 or py>310]
+  skip: True # [py<37 or s390x]
   missing_dso_whitelist:
-    - '*/RDKit*dll'  # [win]
+    - '*/RDKit*dll'                     # [win]
```

Comment:
Isn't it weird that this is whitelisted like this?

Reply:
I think it can be related to this https://github.com/AnacondaRecipes/boost-feedstock/pull/16/files#diff-270cf27907bf32793701614a19a54ec06c7dac8f9bd96790c3ee9348ac0ea347R33.
@M-Waszkiewicz-Anaconda Should '*/RDKit*dll' be removed from `missing_dso_whitelist`  now?

Reply:
Looking at the logs I can say that the issue was resolved by removing this `missing_dso_whitelist`:
`INFO (rdkit,Library/lib/RDKitTautomerQuery.dll): Library\bin\RDKitMolStandardize.dll found in this package`